README.md

```diff
@@ -34,6 +34,13 @@ app.use('/pdf', (req, res) => {
 })
 ```
 
+### Configuring Puppeteer
+Pass args to the Puppeteer launch as follows. See [Puppeteer launch options](https://github.com/GoogleChrome/puppeteer/blob/v1.18.1/docs/api.md#puppeteerlaunchoptions) for more info
+```javascript
+const puppeteerArgs = { args: ['--no-sandbox', '--disable-setuid-sandbox'] }
```

Comment:
probably worth defaulting this and allowing the user to install change them if they want to